include/seastar/http/client.hh

```diff
@@ -93,6 +94,64 @@ private:
     future<reply> recv_reply();
 };
 
+/**
+ * \brief Class client wraps communications using HTTP protocol
+ *
+ * The class allows making HTTP requests and handling replies. It's up to the caller to
+ * provide a transport, though for simple cases the class provides out-of-the-box
```

Comment:
What does "it's up to the caller to provide a transport" mean? It seems an address is given an the transport is TCP to that address, no?